src/calchartdoc.cpp

```diff
@@ -279,6 +281,25 @@ wxInputStream& CalChartDoc::LoadObject(wxInputStream& stream)
 }
 #endif
 
+bool CalChartDoc::exportViewerFile(const wxString& filepath) {    
+    JSONElement mainObjectElement = JSONElement::makeObject();
+    
+    JSONDataObjectAccessor mainObject = mainObjectElement;
```

Comment:
`mainObjectElement` is a `JSONElement`, which can have one of several underlying types. In this case, I know for sure that the underlying type is a JSON Object, which I can gain access to using an Object Accessor. For this reason, I make a `JSONDataObjectAccessor` called `mainObject`. I should go through and rename it to `mainObjectAccessor` to make that more clear here